examples/phi3/phi3_tuning.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Instead of adding a config example, could we just add the `tune-session-params` command line example?